### PR TITLE
added simple graphics

### PR DESCRIPTION
**Pull request by deGekata**

### CODE REVIEW

**Comment by dainbow** [comment]
Fix naming

File: geometry/Line.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include "geometry/Line.hpp"
+#include "geometry/Geometry.hpp"
```
Comment (dainbow) [inline geometry/Line.cpp:2]: <>

File: geometry/Plane.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#include "geometry/Plane.hpp"
+#include "geometry/Operations.hpp"
+namespace geometry {
```
Comment (dainbow) [inline geometry/Plane.cpp:3]: Blank line

File: include/geometry/GeomVector.hpp [hunk 3]
```diff
@@ -0,0 +1,257 @@
+#ifndef GEOMETRY_VECTOR_HPP
+#define GEOMETRY_VECTOR_HPP
+#include <geometry/Float.hpp>
```
Comment (dainbow) [inline include/geometry/GeomVector.hpp:3]: Blank line

File: include/geometry/Point.hpp [hunk 4]
```diff
@@ -0,0 +1,128 @@
+#ifndef GEOMETRY_POINT_HPP
+#define GEOMETRY_POINT_HPP
+#include <cassert>
+#include <cstdint>
+#include <cmath>
+
+namespace geometry {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wconversion"
```
Comment (dainbow) [inline include/geometry/Point.hpp:9]: Ban

File: geometry/GeomVector.cpp [hunk 5]
```diff
@@ -1,6 +1,7 @@
-#include "geometry/GeomVector.hpp"
-#include "geometry/Point.hpp"
-#include "geometry/Float.hpp"
+#include <geometry/geomVector.hpp>
```
Comment (XDLoLiK) [inline geometry/GeomVector.cpp:1]: snake case.

File: include/graphics/aWindow.hpp [hunk 6]
```diff
@@ -1,14 +1,15 @@
...
+
+#include "geometry/geometry.hpp"
+
+#include <graphics/color.hpp>
+#include <graphics/event.hpp>
+#include <graphics/aObject.hpp>
+#include <graphics/aText.hpp>
+#include <graphics/aRenderSurface.hpp>
+#include <graphics/surface.hpp>
 namespace igraphicslib {
```
Comment (XDLoLiK) [inline include/graphics/aWindow.hpp:13]: blank line.

File: include/geometry/rect.hpp [hunk 7]
```diff
@@ -1,11 +1,13 @@
 #ifndef GEOMETRY_RECT_HPP
 #define GEOMETRY_RECT_HPP
-#include "geometry/Point.hpp"
+#include <geometry/point.hpp>
 
 namespace geometry {
+
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wconversion"
 #pragma clang diagnostic ignored "-Wsign-conversion"
```
Comment (XDLoLiK) [inline include/geometry/rect.hpp:9]: No pragmas. Will leave it for now, but fix in the future.

**Review by XDLoLiK — CHANGES_REQUESTED** [review]

**Pull request merged**